src/lib/viewers/controls/model3d/AnimationClipsControl.scss

```diff
@@ -0,0 +1,23 @@
+.bp-AnimationClipsControl {
+    position: relative;
+    display: flex;
+    align-items: center;
+
+    .bp-SettingsFlyout {
+        right: auto;
+        left: 0;
+        max-height: 200px;
+        overflow-x: hidden;
+        overflow-y: auto;
+    }
+}
+
+.bp-AnimationClipsControl-radioItem {
+    .bp-SettingsRadioItem-value {
```

Comment:
Could we set these styles on the parent class, instead of selecting an inner class?